release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1089,6 +1089,52 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, if multiple routes were specified in the same certificate or if a route specified the default certificate as a custom certificate, and HTTP/2 was enabled on the router, an HTTP/2 client could perform connection coalescing on routes. Clients, such as a web browser, could re-use connections and potentially connect to the wrong backend server. With this release, the {product-title} router now checks when the same certificate is specified on more than one route or when a route specifies the default certificate as a custom certificate. When either one of these conditions is detected, the router configures the HAProxy load balancer so to not allow HTTP/2 client connections to any routes that use these certificate.(link:https://issues.redhat.com/browse/OCPBUGS-29373[*OCPBUGS-29373*])
+
+* Previously, if you configured a deployment with the `routingViaHost` parameter set to `true`, traffic failed to reach the IPv6 `ExternalTrafficPolicy=Local` load balancer service. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-27211[*OCPBUGS-27211*])
+
+* Previously, a pod selected by an `EgressIp` object that was hosted on a secondary network interface card (NIC) caused connections to node IP addresses to timeout. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-26979[*OCPBUGS-26979*])
+
+* Previously, a leap file package that the {product-title} Precision Time Protocol (PTP) Operator installed could not be used by the `ts2phc` process because the package expired. With this release, the leap file package is updated to read leap events from Global Positioning System (GPS) signals and update the offset dynamically so that the expired package situation no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-25939[*OCPBUGS-25939*])
+
+* Previously, pods assigned an IP from the pool created by the Whereabouts CNI plugin were getting stuck in `ContainerCreating` state after a node force reboot. With this release, the Whereabouts CNI plugin issue associated with the IP allocation after a node force reboot is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-24608[OCPBUGS-24608])
+
+* Previously, there was a conflict between two scripts on {product-title} in IPv6, including single and dual-stack, deployments. One script set the hostname to a fully qualified domain name (FQDN) but the other script might set it to a short name too early. This conflict happened because the event that triggered setting the hostname to FQDN might run after the script that set it to a short name. This occurred due to asynchronous network events.
```

Comment:
```suggestion
* Previously, there was a conflict between two scripts on {product-title} in IPv6, including single and dual-stack, deployments. One script set the hostname to a fully qualified domain name (FQDN), but the other script might set it to a short name too early. This conflict happened because the event that triggered setting the hostname to FQDN might run after the script that set it to a short name. This occurred due to asynchronous network events.
```

Reply:
We might want to add "With this release, this issue is fixed..."

Reply:
Missing bug link.

Reply:
:heavy_check_mark: